app/javascript/src/components/Notes/NoteDropDown.jsx

```diff
@@ -14,7 +14,11 @@ const NoteDropDown = () => {
       >
         <Menu>
           <MenuItem.Button key={0}>Edit</MenuItem.Button>
-          <MenuItem.Button key={1} style="danger">
+          <MenuItem.Button
+            key={1}
```

Comment:
Why's this key prop required over here?

Reply:
Added it when I saw it on neetoui docs , but there they were looping through an array. Here we don't require key attribute , understood.